app/Console/Kernel.php

```diff
@@ -28,7 +29,8 @@ class Kernel extends ConsoleKernel
         EndedProject::class,
         FixedBudgetProject::class,
         NotificationToProjectTeamMembersToUpdateEffortOnGoogleChat::class,
-        JobExpiredEmailToHr::class
+        JobExpiredEmailToHr::class,
+        SystemReviewQuaterly::class
```

Comment:
Please update all places accordingly
```suggestion
        QuarterlyReviewSystemForEmployee::class
```